modape/modis/window.py

```diff
@@ -138,8 +138,11 @@ def generate_mosaics(self,
             labels = None
             force_doy = True
 
+        output_res = [None, None]
```

Comment:
Why don't we use `output_res = []`? or maybe even better with a tuple `output_res = ()`.

This way we can do a simple `if aoi is not None and output_res:` in line 226.

Reply:
The
`output_res = [None, None]`
was in the original code... Also, my guess is that it matches with the default in PyGDAL's warp signature.

Reply:
Right, that's why we don't use it 😄

Reply:
The `[None, None]` array was there in case of no resolution specified; we will of course still support that scenario. Meaning the `[None, None]`, just like in the original code, still makes sense to me. Take a look at this code that is invoked from generate_mosaics:
https://github.com/WFP-VAM/modape/blob/5dd5fa7b942b0a6f6a946b353050d50b9abda9b2/modape/modis/window.py#L352-L376

This code expects an argument passed in that can by indexed by 0 and 1. Also, check out the defaults for xRes and yRes in WarpOptions: https://gdal.org/python/osgeo.gdal-module.html#WarpOptions

All-in-all: `[None, None]` is fine with me and, in fact, I like it better than a tuple as I generally use tuples for heterogeneous sets. Whereas a list would imply (to me at least) a homogeneous set. Also, conceptually, tuples are less widespread (for example: JavaScript has no equivalent).

I suggest we go with the code as proposed.